memcached.c

```diff
@@ -148,10 +147,7 @@ conn **conns;
 struct slab_rebalance slab_rebal;
 volatile int slab_rebalance_signal;
 #ifdef EXTSTORE
-/* hoping this is temporary; I'd prefer to cut globals, but will complete this
- * battle another day.
- */
-void *ext_storage = NULL;
+static storage_engine *engine;
```

Comment:
ext_storage was only used in one place in items.c. I'd rather not regress to more globals by removing the worker thread reference and using this more